src/code/templates/fv-egg-game.js

```diff
@@ -1343,7 +1343,7 @@ export default class FVEggGame extends Component {
       return authoredDrakes;
 
     // challengeType === 'match-target'
-    const targetDrakeCount = authoredChallenge.targetDrakes.length;
+    const targetDrakeCount = authoredChallenge.numTrials || authoredChallenge.targetDrakes.length;
```

Comment:
Awesome. It's nice to get rid of the `targetDrakes: [{},{},{}]`-style authoring.